server/methods/browseChannels.js

```diff
@@ -24,6 +24,10 @@ const sortChannels = function(field, direction) {
 
 const sortUsers = function(field, direction) {
 	switch (field) {
+		case 'email':
+			return {
+				emails: direction === 'asc' ? 1 : -1,
```

Comment:
please, check if this line shouldn't  be
```suggestion
				'emails.address': direction === 'asc' ? 1 : -1,
```

thanks

Reply:
Yup, better with your suggestion. Committing, thank you.

Reply:
Changes applied, ready for re-review